.github/workflows/publish.yml

```diff
@@ -12,18 +12,17 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
 
       - name: Set up Python
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v2
```

Comment:
Could you inform us what are the diff. in v1 and v2? I'm sure it's a good thing. Don't bother to look into it too much as long as it passes.